bundles/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/internal/GPIOBindingConstants.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
```

Comment:
year in copyright can probably be fixed by running `mvn spotless:apply`
```suggestion
 * Copyright (c) 2010-2021 Contributors to the openHAB project
```